src/tests/ftest/datamover/dm_dst_create.py

```diff
@@ -274,14 +270,57 @@ def verify_usr_attr(self, cont):
                       len(attrs.keys()), attrs.items())
 
     @avocado.fail_on(DaosApiError)
-    def test_dm_dst_create_dcp(self):
+    def test_dm_dst_create_dcp_posix_dfs(self):
         """
         Test Description:
-            Verifies destination container creation, including
+            Verifies destination container creation
+            for DFS API, including
             container properties and user attributes.
 
         :avocado: tags=all,full_regression
         :avocado: tags=datamover,dcp
-        :avocado: tags=dm_dst_create,dm_dst_create_dcp
+        :avocado: tags=dm_dst_create_dcp_posix_dfs
```

Comment:
Each of these test cases should also be tagged with "dm_dst_create" to group them together. Then, you can run all tests in this file with the tag "dm_dst_create"
```suggestion
        :avocado: tags=dm_dst_create,dm_dst_create_dcp_posix_dfs
```